bundledata.go

```diff
@@ -50,6 +50,10 @@ type BundleData struct {
 	// the bundle chooses charms.
 	Series string `bson:",omitempty" json:",omitempty" yaml:",omitempty"`
 
+	// Base holds the default base to use when the bundle chooses
+	// charms. Base should overwrite Series
```

Comment:
We should error if both are provided. Otherwise the user may have unexpected results.

Reply:
Resolved in latest commit